packages/clerk-js/package.json

```diff
@@ -44,10 +44,12 @@
     "@popperjs/core": "^2.4.4",
     "browser-tabs-lock": "^1.2.15",
     "classnames": "^2.3.1",
+    "copy-to-clipboard": "^3.3.2",
```

Comment:
❓ How does this affect the bundle size? Is it a big package to add?

Reply:
This lib should add about ~1kb to the final bundle, so it's okish 

Another alternative would be to extract the logic we need from that repo and make a custom hook instead, but this can be done as an optimization in the future

Reply:
Not suggesting that we change anything in this PR, but:

- It looks like the [method](https://github.com/sudodoki/copy-to-clipboard/blob/193826f9859923d4e83c35093f3c436be00a162b/index.js#L79) that's used by the copy-to-clipboard package is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#browser_compatibility). 
- There is a newer [clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) which is currently the recommended way to "copy to clipboard".
- Looks like [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API#browser_compatibility) won't be an issue.

Reply:
Will handle in a subsequent PR

Just re-used the same `copy-to-clipboard` dep Chakra uses in the dashboard.